app/allocation/views/view.njk

```diff
@@ -77,79 +77,81 @@
     {% endif %}
   </section>
 
-  <section class="govuk-!-margin-top-8">
-    <h2 class="govuk-heading-m govuk-!-margin-bottom-1">
-      {{ t("allocation::view.people.heading") }}
-    </h2>
-
-    <h3 class="govuk-heading-s">
-      {% if remainingCount === 0 %}
-        <span class="app-icon app-icon--tick app-green">{{ t("allocation::view.people.complete") }}</span>
-      {% elif canAccess('allocation:person:assign') %}
-        {{ t("allocation::view.people.progress", {
-          count: remainingCount,
-          context: "remaining"
-        }) }}
-      {% else %}
-        {{ t("allocation::view.people.progress", {
-          count: totalCount,
-          addedCount: addedCount
-        }) }}
-      {% endif %}
-    </h3>
-
-    {% if canAccess('allocation:person:assign') and remainingCount %}
-      <p>
-        {{ t("allocation::view.people.special_vehicle_instruction", {
-          href:"/move/new"
-        }) | safe }}
-      </p>
-
-      {{ govukButton({
-        href: "/move/" + unassignedMoveId + "/assign",
-        html: t("actions::add_person", {
-          context: "another" if addedCount !== 0
-        })
-      }) }}
-    {% endif %}
-  </section>
+  {% if allocation.status != "cancelled" %}
```

Comment:
I don't expect anything in this PR, but it makes me sad that the current state of the browser tests being E2E tests mean they're too slow to verify these screen behave as we'd expect, and that there aren't any component tests we could leverage for similar testability.

Reply:
It would also be nice if we had unit tests for the views and the conditionals were handled in the view/controller code ;)